### PR TITLE
Javalab: Exit dropdowns using Escape key

### PR DESCRIPTION
**Pull request by sanchitmalhotra126**
Quick win - adds an onKeyDown handler to our various dropdowns to close dropdown when the escape key is pressed.

https://user-images.githubusercontent.com/85528507/219129761-32f2ce31-1ef0-4e5f-a976-52590c1120ac.mov

## Links

https://codedotorg.atlassian.net/browse/SL-108

## Testing story

Tested locally.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/javalab/JavalabEditorTabMenu.jsx [hunk 1]
```diff
@@ -23,10 +23,6 @@ export class JavalabEditorTabMenu extends Component {
     orderedTabKeys: PropTypes.array
   };
 
-  state = {
-    dropdownOpen: false
-  };
-
```
Comment (sanchitmalhotra126) [inline apps/src/javalab/JavalabEditorTabMenu.jsx:29]: Not related to the current change, but I noticed this state value was unused

File: apps/src/javalab/Backpack.jsx [hunk 2]
```diff
@@ -217,6 +217,12 @@ class Backpack extends Component {
     }
   };
 
+  handleKeyDown = event => {
```
Comment (molly-moen) [inline apps/src/javalab/Backpack.jsx]: could we turn this into a helper component that is just a div that is closed on 'escape' and you can put any children inside it?

Comment (sanchitmalhotra126) [reply apps/src/javalab/Backpack.jsx]: Hmm yeah, I was thinking of creating a HOC sort of like `onClickOutside` at first but that seemed like a little much for this use case. However, just creating a wrapper component seems to be straightforward, I can try that

Comment (sanchitmalhotra126) [reply apps/src/javalab/Backpack.jsx]: Reworked this to use a wrapper component - let me know if that looks better!

**Review by molly-moen — APPROVED** [review]
nice!

**Pull request merged**